src/UserActivityFeedEventSchemas/AcademicProfileTagDeletedEventSchema.js

```diff
@@ -0,0 +1,34 @@
+module.exports = (conn) => {
+  const AcademicProfileTagDeletedEventSchema = new conn.Schema({
+    createdBy: {
```

Comment:
I've seen a commit updating this name in the changeset. I would suggest to remove it and also consider it to be undone and keep the name as `removedBy` or similar, I think mixing a creation property for a deletion document can lead to confusion